python/ruff/__main__.py

```diff
@@ -31,4 +32,5 @@ def find_ruff_bin() -> Path:
 
 if __name__ == "__main__":
     ruff = find_ruff_bin()
-    sys.exit(os.spawnv(os.P_WAIT, ruff, ["ruff", *sys.argv[1:]]))
+    completed_process = subprocess.run([ruff, *sys.argv[1:]])
+    sys.exit(completed_process.returncode)
```

Comment:
I feel like this was changed to `os.spawnv` intentionally. Can you look through the Git history to figure out why it was written this way initially?

Reply:
This is oldest comment i can find: https://github.com/astral-sh/ruff/pull/687/files#r1020543897 . I've used `execv` on unix in monotrail when i wanted python to take over the process completely (vs. a subcommand), but i'm rather confident changing away at least from `os.spawnv` given that the docs recommend using subprocess instead (https://docs.python.org/3/library/os.html#os.spawnv)